front/public/locales/en/infraEditor.json

```diff
@@ -1,162 +1,2251 @@
 {
   "ApplicableDirections": {
-    "description": "Used to determine the direction of application of certain objects.",
-    "title": "Applicable directions"
+    "description": "This is used to know the direction of application of certain objects",
+    "enum": [
```

Comment:
Hm, it seems like this adds a bunch of non-translatable material in the translation files.

How bad would it be to add another `jq` filter to the script to only retain `title` and `description` properties? This would make translators' jobs easier.